code/datums/mood.dm

```diff
@@ -0,0 +1,471 @@
+#define MINOR_INSANITY_PEN 5
+#define MAJOR_INSANITY_PEN 10
+#define MOOD_CATEGORY_NUTRITION "nutrition"
+#define MOOD_CATEGORY_AREA_BEAUTY "area_beauty"
+
+/**
+ * Mood datum
+ *
+ * Contains the logic for controlling a living mob's mood and sanity.
+ */
+/datum/mood
+	/// Weakref to the parent (living) mob
+	var/datum/weakref/parent
+
+	/// Mob's mood
```

Comment:
not very descriptive, whats the value type? if its a number, what values correspond to happy and what values correspond to sad?